packages/block-template/templates/custom-element/package.json

```diff
@@ -25,12 +25,12 @@
     "serve": "node_modules/.bin/block-scripts serve"
   },
   "dependencies": {
-    "@blockprotocol/graph": "0.0.11",
+    "@blockprotocol/graph": "0.0.12",
     "lit": "^2.2.5"
   },
   "devDependencies": {
     "block-scripts": "0.0.14",
-    "mock-block-dock": "0.0.17",
+    "mock-block-dock": "0.0.18-canary.1",
```

Comment:
this can be `0.0.18`

Reply:
Thanks. I planned to update once approved. Will update now